draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -62,9 +59,10 @@ The widespreading of HTTP as a distributed computation protocol
 requires an explicit way of communicating service status and
 usage quotas.
 
-This was partially addressed with the `Retry-After` header field
+This was partially addressed by the `Retry-After` header field
 defined in {{!SEMANTICS=I-D.ietf-httpbis-semantics}} to be returned in
-`429 Too Many Requests` or `503 Service Unavailable` responses.
+`429 Too Many Requests` (see {{?STATUS429=RFC6525}})
+or `503 Service Unavailable` responses.
```

Comment:
No link for 503?

Reply:
503 is in semantics... I think it's redundant but if you think it helps I can add it.

Reply:
Just mentioned it for the sake of consistency with 429, but indeed not needed.